dms_version/models/dms_file.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2017-2020 MuK IT GmbH
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class DmsFile(models.Model):
+    _inherit = "dms.file"
+
+    version_ids = fields.One2many(
+        comodel_name="dms.version",
+        inverse_name="file_id",
+        string="Versions",
+        readonly=True,
+    )
+    count_versions = fields.Integer(
+        compute="_compute_count_versions", string="Count Versions"
+    )
+
+    @api.model
+    def _check_context_no_versioning(self):
+        return self.env.context.get("migration") or self.env.context.get(
```

Comment:
Why is this variable used? 
If it is specific to this module, could you do the same for the other variable?
```suggestion
        return self.env.context.get("dms_migration") or self.env.context.get(
```